site/gatsby-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    siteMetadata: {
+        title: `Twenty Twenty Site`,
+        description: `Gatsby site for Twenty Twenty Gatsby Theme.`,
+        author: `@sagarnasit`,
+        wordPressUrl: `wordpress.com`
```

Comment:
Is this correct url? Should this be linked to the actual site where WordPress instance is deployed?

For now maybe we could add local setup url / demo env, and add a comment informing the use of `wordPressUrl `

Reply:
This is just for example. Replace this to local wordpress site.